### PR TITLE
fix(rnafusion): mapped reads percentage calculation (patch)

### PR DESCRIPTION
**Pull request by fevac**
### Fixed

- Mapped reads percentage was incorrect in the delivery report as it was dividing aligned reads (in read pairs) by total reads (in number of reads). Solved by multiplying aligned reads x 2.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b fix_mapped_reads_perc -a
    ```

### How to test

- [x] cg generate delivery-report <case_id> --dry-run
- [x] cg upload delivery-report-to-scout  <case_id>. Make sure that `Mappade sekvenser` has the right value.

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by fevac** [comment]
ready for review

File: cg/meta/report/rnafusion.py [hunk 1]
```diff
@@ -55,7 +47,9 @@ def get_sample_metadata(
             input_amount=self.lims_api.get_latest_rna_input_amount(sample_id=sample.internal_id),
             insert_size=None,
             insert_size_peak=None,
-            mapped_reads=sample_metrics.reads_aligned / sample_metrics.before_filtering_total_reads,
+            mapped_reads=sample_metrics.reads_aligned
```
Comment (henrikstranneheim) [inline cg/meta/report/rnafusion.py:50]: Should reads_aligned be renamed to reflect that it is read pairs and not single reads - to avoid future confusion.

**Review by henrikstranneheim — APPROVED** [review]
Nice catch 👍

File: cg/meta/report/rnafusion.py (hunk 1 above)
Comment (fevac) [reply cg/meta/report/rnafusion.py:50]: these are standard names given by multiqc. So I don't think its a great idea to change them. It is nice that the metrics collected by multiqc always get assigned the same names regardless of pipeline, etc. I do agree that several of these names are confusing. But maybe something to think about in the future? I don't think I would do it now

Comment (henrikstranneheim) [reply cg/meta/report/rnafusion.py:50]: Ok, let's leave it for now 👍. If you want to you could make an issue on multiqc to clarify the variable name.

**Comment by fevac** [comment]
tests passing

**Comment by fevac** [comment]
- Deployed in stage:
image

- Deployed in prod:
image

**Pull request merged**